README.md

```diff
@@ -35,7 +35,22 @@ B. Simplified versions of plots in the main QC report.
 
 The `figures` and `tables` folders contain the most up-to-date version of each of the figures and tables, respectively.
 The `scripts` directory contains all scripts used to create the figures and tables.
-See the [`README` for the `scripts` directory](./scripts/README.md) for more information on generating figures.
+See the [`README` for the `scripts` directory](./scripts/README.md) for more information on figure and table scripts.
+
+To generate all figures and tables, run the script [`generate-figures-tables.sh`](generate-figures-tables.sh) as:
+
+```sh
+bash generate-figures-tables.sh
+```
+
+Note that this script assumes that the `s3_files` directory has been populated with relevant data files.
+These files can be obtained by first running the figure setup scripts, which currently require S3 bucket access as a Data Lab member.
+Setup scripts can be run as:
+
+```sh
+Rscript scripts/figure_setup/sync-metadata.R
+Rscript scripts/figure_setup/sync-data-files.R
```

Comment:
Should we add `op run --` here?

Reply:
I think if we're keeping the S3 text for now as "you need to be a data lab member," then yeah it's fine to add this. 
I'll open another issue "before we go public, clean up this README to be more public-facing".